config/routes.rb

```diff
@@ -106,7 +106,7 @@
     scope module: "lead_providers", path: "lead-providers" do
       resources :lead_provider_users, only: %i[edit update], path: "users"
     end
-    resources :induction_coordinators, only: %i[edit update], path: "induction-coordinators"
+    resources :induction_coordinators, only: %i[index edit update], path: "induction-coordinators"
```

Comment:
Huh, did we have dead routes before this PR? Oh well, they aren't dead anymore.

Reply:
Yes, I have to add them in the earlier pr to test the helper